src/settings.yml.sample

```diff
@@ -117,3 +117,76 @@ pinanas:
 
       # Send emails as. Type: string. Example: "contact@{{ pinanas.domain }}".
       sender:
+
+
+  applications:
+    #
+    # === SELECT YOUR APPS === #
+    #
+    # PiNANAS includes following apps by default:
+    # - DHCPD
+    # - Traefik
+    # - Fail2ban
+    # - Authelia
+    #
+    # Optional applications can also be activated according to your needs. For a brief description of theses apps, go read the chapter "APPS DESCRIPTION" at the end of this file.
+    # To activate one, please uncomment wanted application bellow.
+    # (by default, optionnal apps are not activated)
+    #
+  #  - adguardhome
+  #  - heimdall
+  #  - jellyfin
+  #  - netdata
+  #  - nextcloud
+
+#
```

Comment:
Better create a dedicated documentation, e.g. `docs/enabling-and-disabling-applications.md` and link to it from this file. This allows formatting, referral and discovery.